packages/labs/ssr/README.md

```diff
@@ -43,15 +51,99 @@ export const renderTemplate = (someData) => {
 // Example: server.js:
 
 import {renderModule} from 'lit-ssr/lib/render-module.js';
-// Execute the above `renderTemplate` in a sandboxed VM with a minimal DOM shim
+
+// Execute the above `renderTemplate` in a separate VM context with a minimal DOM shim
 const ssrResult = await (renderModule(
-  './render-template.js',  // Module to load in SSR sandbox
+  './render-template.js',  // Module to load in VM context
   import.meta.url,         // Referrer URL for module
   'renderTemplate',        // Function to call
   [{some: "data"}]         // Arguments to function
 ) as Promise<Iterable<unknown>>);
 
-for (let text of ssrResult) {
-  console.log(text);
-}
+// ...
+
+context.body = Readable.from(ssrResult);
 ```
+
+## Client usage
+
+### Hydrating lit templates
+
+In order to "hydrate" lit templates, which is the process of having lit re-associate the expressions of a lit template with which nodes they should update in the DOM, the `experimental-hydrate` module is provided in the `lit` package. Prior to updating a server-rendered container using `render`, you should first call `hydrate` on that container using the same template and data that was used to render on the server:
```

Comment:
You might just want to split this into two sentences: "Hydration is the process.... . In order to hydrate Lit templates..."

Reply:
Done